internal/util/util.go

```diff
@@ -335,9 +177,9 @@ func GenerateBundleName(bdName, hash string) string {
 	return fmt.Sprintf("%s-%s", bdName, hash)
 }
 
-// SortBundlesByCreation sorts a BundleList's items by it's
+// SortBundleDeploymentsByCreation sorts a BundleDeploymentList's items by its
 // metadata.CreationTimestamp value.
-func SortBundlesByCreation(bundles *rukpakv1alpha1.BundleList) {
+func SortBundleDeploymentsByCreation(bundles *rukpakv1alpha2.BundleDeploymentList) {
```

Comment:
Similar question as above. This function was used in the context of multiple `Bundles` existing for a BD. Now that we no longer have a Bundle API, I wonder if we still need/use this function?

Reply:
My bad, missed cleaning this up. Wasn't being used anywhere.